source/nodejs/adaptivecards-designer/src/card-designer-surface.ts

```diff
@@ -260,52 +260,52 @@ export class CardDesignerSurface {
     private renderCard() {
         this._cardHost.innerHTML = "";
 
-        if (this.card) {
-            if (this.onCardValidated) {
-                this.onCardValidated(this._lastParseErrors, this.card.validateProperties());
+        if (this.onCardValidated) {
```

Comment:
Unrelated but should probable be addressed; Input Validation as a feature is not expected to land until our next big release (1.3); we had enabled it in master thinking it was closer to ship but since it's not can we disable the feature again please so we don't end up confusing customers? Thanks!

Reply:
There is no change to what's already in master here. Input validation (as currently implemented) is and has been disabled by default in JS.